projects/demo/src/modules/components/input-phone-international/index.ts

```diff
@@ -23,6 +27,10 @@ import {InheritedDocumentation} from '../abstract/inherited-documentation';
         InheritedDocumentation,
         ReactiveFormsModule,
         TuiDemo,
+        TuiDocControl,
+        TuiDocDropdown,
+        TuiDocIcons,
+        TuiDocTextfield,
```

Comment:
Please also remove `tuiProvide(ABSTRACT_PROPS_ACCESSOR, PageComponent),` and `extends AbstractExampleTuiControl` from this file

Reply:
Removed